include/spdk/env.h

```diff
@@ -126,6 +131,8 @@ void spdk_memzone_dump(FILE *f);
 
 struct spdk_mempool;
 
+#define SPDK_MEMPOOL_CACHE_MAX_SIZE 512
```

Comment:
Jim Harris and I discussed this.  The plan is to include a compile time assert in env.c that insures this definition, as well as a couple of others, don't get out of sync with what's in DPDK.